src/rust/ensogl/lib/core/src/system/gpu/data/gl_enum.rs

```diff
@@ -31,13 +31,13 @@ pub mod traits {
     use super::*;
 
     /// Methods for every object which implements `Into<GlEnum>`.
-    pub trait IntoGlEnum {
+    pub trait ToGlEnum {
         /// Converts the current value to `GlEnum`.
-        fn into_gl_enum(&self) -> GlEnum;
+        fn to_gl_enum(&self) -> GlEnum;
     }
 
-    impl<T> IntoGlEnum for T where for<'a> &'a T:Into<GlEnum> {
-        fn into_gl_enum(&self) -> GlEnum {
+    impl<T> ToGlEnum for T where for<'a> &'a T:Into<GlEnum> {
+        fn to_gl_enum(&self) -> GlEnum {
             self.into()
```

Comment:
Why this change? Is it something clippy complained about?

I'm just curious.

Reply:
The "into_*" and "to_*" methods are supposed to follow the convention about self type, see: https://rust-lang.github.io/rust-clippy/master/#wrong_self_convention

Reply:
This is a good change. Im wondering why it was ever named `into_`